.circleci/config.yml

```diff
@@ -234,3 +243,17 @@ jobs:
       - publish_docker_images:
           repo: opentelemetry-collector-dev
           tag: ${CIRCLE_SHA1}
+
+  windows-test:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run:
+          shell: powershell.exe
+          command: choco install make
+      - run:
+          name: Unit tests
+          shell: powershell.exe
+          command: make test
```

Comment:
Installing `make` on Windows isn't enough. Most commands in our makefile won't run on Windows. You'd need to download Cygwin or something similar to get this working.

Testing locally with Cygwin I could get this working with the following (converted to an untested CircleCI step):

```yaml
- run:
    shell: powershell.exe
    command: |
      choco install --no-progress --yes cygwin
      choco install --no-progress make mingw64-x86_64-gcc-core --source=cygwin
- run:
    shell: C:/tools/cygwin/bin/bash.exe --noprofile --norc -eo pipefail
    command: |
      alias gcc=x86_64-w64-mingw32-gcc
      make test
```

The downside of this is that it requires downloading cygwin & mingw64-x86_64-gcc-core which is ~0.5GB uncompressed. The other option would be to just execute `go test` commands directly via powershell, which might be simpler.

Reply:
Done.